activerecord/lib/arel/visitors/oracle12.rb

```diff
@@ -18,9 +18,9 @@ def visit_Arel_Nodes_SelectStatement(o, collector)
         end
 
         def visit_Arel_Nodes_SelectOptions(o, collector)
-          collector = maybe_visit o.offset, collector
-          collector = maybe_visit o.limit, collector
```

Comment:
These are semantically important (visit always replaces the collector), although I don't think there's actually any code that relies on it